src/cmd-koji-upload

```diff
@@ -406,6 +406,32 @@ class _KojiBase():
         return session
 
 
+class Search(_KojiBase):
+    """
+    Search for builds
+    """
+
+    def __init__(self, profile):
+        """
+        Creates a new instance for search.
+
+        :param profile: Koji profile name in /etc/koji.conf.d
+        :type str
+        """
+        super().__init__(profile)
+
+    def get_state(self, nvr):
+        """
+         Return the build state.
```

Comment:
Is there a link to documentation we can put here that describes what the different states could be?